src/components/Home/index.js

```diff
@@ -74,6 +78,17 @@ class Home extends Component {
         </View>
       );
     }
+    if (isEmptyEventsList) {
+      return (
+        <View style={styles.container}>
+          <Headbar title="TRANG CHỦ" />
+          <Searchbar />
+          <View style={styles.emptyListContainer}>
+            <Text style={styles.emptyListText}>Chưa có sự kiện nào</Text>
+          </View>
+        </View>
+      );
+    }
```

Comment:
View, Header and SideBar are hte same in 2 cases. Only conditional render FlatList.

Reply:
ok